starwars/src/components/CardList.js

```diff
@@ -0,0 +1,13 @@
+// your components will all go in this `component` directory.
+// feel free to change this component.js into TodoList.js
+import React from "react";
+import Card from "./Card";
+
+function CardList(props) {
+  return <ul>{props.cards.map((card, i) => <Card
+    card={card}
+    key={props.cards[i]}/>)}
```

Comment:
We talked about this in our one on one, but a key needs to be a unique identifier. Using the index of the array is a clever way to handle this, but that means you just need `{i}` not `{props.cards[i]}`